README.md

```diff
@@ -164,7 +164,7 @@ additional methods:
 
 ### Set
 
-Represents a collection like [Array](#Array) except without ordering or duplicate values.
+Represents a collection like [Array](#Array) except without ordering or duplicate values. It is possible to assign a comparer function in the opts, to be used for determining uniqueness.
```

Comment:
Might also be worth mentioning that the items added to the set can be observables (and probably should be, otherwise you might as well just use a computed or something). They are resolved to the final values whenever they change. 

This is also a bit of a complication in regards to this pull request, because the uniqueness check is only done when they are added, not when they update.